Codigo/Protocol.bas

```diff
@@ -3101,6 +3101,7 @@ Private Sub HandleWork(ByVal UserIndex As Integer)
 112                 Call WriteWorkRequestTarget(UserIndex, Skill)
 
 114             Case Ocultarse
+                    Packet_ID = PacketNames.Hide
```

Comment:
Why do we overwrite the `Packet_ID` here, it's assigned in 3083 and never used. Is this assignment necessary?

Reply:
Remove `Packet_ID = PacketNames.Hide`

And just use 
```
 If Not verifyTimeStamp(PacketCounter, .PacketCounters(PacketNames.Hide), .PacketTimers(PacketNames.Hide), .MacroIterations(PacketNames.Hide), userindex, "Ocultar", PacketTimerThreshold(PacketNames.Hide), MacroIterations(PacketNames.Hide)) Then Exit Sub
 ```